panel/tests/test_links.py

```diff
@@ -207,11 +207,11 @@ def test_widget_bkplot_link(document, comm):
     assert scatter.glyph.fill_color == '#ff00ff'
     
     code = """
-    value = source['color'];
+    var value = source['color'];
     value = value;
     value = value;
```

Comment:
@philippjfr, that's an odd bit of code ^^^.

Reply:
Agreed, it's auto-generated and those empty value assignments are placeholders for possible transforms that may be applied at those stages but are simply no-ops in this case.

Reply:
> it's auto-generated

OK, that's the missing bit.

Reply:
And it's a test file. I didn't notice that before.